.babelrc

```diff
@@ -4,6 +4,7 @@
     "react"
   ],
   "plugins": [
-    "transform-object-rest-spread"
+    "transform-object-rest-spread",
+    "transform-class-properties"
```

Comment:
99% confident that our consumers will be ok with this change is it ships by default in `backpack-react-scripts` (this is via [`babel-preset-react-app`](https://github.com/facebookincubator/create-react-app/tree/master/packages/babel-preset-react-app)).